src/form/types/types.ts

```diff
@@ -142,6 +143,9 @@ export const NID_VERIFICATION_BUTTON = 'NID_VERIFICATION_BUTTON'
 export const DIVIDER = 'DIVIDER'
 export const HEADING3 = 'HEADING3'
 export const SIGNATURE = 'SIGNATURE'
+export const REDIRECT = 'REDIRECT'
+export const ID_READER = 'ID_READER'
+export const HTTP = 'HTTP'
```

Comment:
@tahmidrahman-dsi should we be adding these types into the common [countryconfig](https://github.com/opencrvs/opencrvs-countryconfig) repo?  What do you think?

Reply:
Yes I think we should, I will open a PR for that

Reply:
https://github.com/opencrvs/opencrvs-countryconfig/pull/359